app/workers/decline_offers_by_default_worker.rb

```diff
@@ -2,11 +2,9 @@ class DeclineOffersByDefaultWorker
   include Sidekiq::Worker
 
   def perform
-    GetApplicationChoicesReadyToDeclineByDefault.call.each do |choice|
-      begin
-        DeclineOfferByDefault.new(application_choice: choice).call
-      rescue StandardError => e
-        Rails.logger.warn "[DBD] ignoring application_choice #{choice.id}: #{e.message}"
+    GetApplicationFormsReadyToDeclineByDefault.call.each do |application_form|
```

Comment:
Nothing to do with this PR but I'm wondering whether we should be using `find_each` here instead of `each` to avoid loading too many model objects into memory at once.

Reply:
Yeah good shout, let's keep that in mind.